misc/config_tools/static_allocators/gpa.py

```diff
@@ -361,10 +361,10 @@ def create_vuart_node(allocation_etree, vm_id, devdict):
 
 def create_native_pci_hole_node(allocation_etree, low_mem, high_mem):
     common.append_node("/acrn-config/hv/MMIO/MMIO32_START", hex(low_mem[0].start).upper(), allocation_etree)
-    common.append_node("/acrn-config/hv/MMIO/MMIO32_END", hex(low_mem[0].end + 1).upper(), allocation_etree)
+    common.append_node("/acrn-config/hv/MMIO/MMIO32_END", hex(low_mem[-1].end + 1).upper(), allocation_etree)
```

Comment:
So `low_mem` and `high_mem` are already sorted here?

Reply:
Yes, see the last time of get_pci_hole_native method.

>     return list(sorted(low_mem)), list(sorted(high_mem))